.eslintrc.cjs

```diff
@@ -1,26 +1,44 @@
 module.exports = {
   root: true,
-  parser: "@typescript-eslint/parser",
   extends: [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
+    "plugin:svelte/recommended",
     "prettier",
   ],
-  plugins: ["svelte3", "@typescript-eslint"],
-  ignorePatterns: ["*.cjs", "static/workers/**/*"],
-  overrides: [{ files: ["*.svelte"], processor: "svelte3/svelte3" }],
-  settings: {
-    "svelte3/typescript": () => require("typescript"),
-  },
+  parser: "@typescript-eslint/parser",
+  ignorePatterns: ["static/workers/**/*"],
+  plugins: ["@typescript-eslint"],
   parserOptions: {
     sourceType: "module",
     ecmaVersion: 2020,
+    extraFileExtensions: [".svelte"],
   },
   env: {
     browser: true,
     es2017: true,
     node: true,
   },
+  overrides: [
+    {
+      files: ["*.svelte"],
+      parser: "svelte-eslint-parser",
+      parserOptions: {
+        parser: "@typescript-eslint/parser",
+      },
+    },
+  ],
+  rules: {
```

Comment:
Was this needed for the migration?

Reply:
Because eslint throw errors which previously were ignored.